pkg/plugin/plugin.go

```diff
@@ -122,7 +122,16 @@ func setupVeth(contNetns ns.NetNS, contIfaceName string, requestedMac string, mt
 	// this we will make sure that both ends of the veth pair will be removed
 	// when the container is gone.
 	err := contNetns.Do(func(hostNetns ns.NetNS) error {
-		hostVeth, containerVeth, err := ip.SetupVeth(contIfaceName, mtu, hostNetns)
+		var containerMac net.HardwareAddr
+		if requestedMac != "" {
+			mac, err := net.ParseMAC(requestedMac)
+			if err != nil {
+				return fmt.Errorf("failed to parse requested MAC  %q: %v", requestedMac, err)
+			}
+			containerMac = mac
+		}
+
+		hostVeth, containerVeth, err := ip.SetupVeth(contIfaceName, mtu, requestedMac, hostNetns)
```

Comment:
Please comment in the commit message how is this related to bump of CNI (if it is). If it is unrelated and needed, please open a separate PR for it

Reply:
Ok, fixed. This is related to the bump of CNI.

Reply:
Sorry for the lack of documentation. Indeed ```SetupVeth``` modification is hard to understand. If you have any other questions, feel free to ask it !

Reply:
Thanks a lot @masap o/